dicoogle/src/main/java/pt/ua/dicoogle/server/users/UserExportPresets.java

```diff
@@ -49,38 +56,34 @@ public static boolean savePreset(String username, String presetName, String pres
             return false;
```

Comment:
I think we should allow the user to update existing presets. Let's discard this check and let the file be overwritten.

This also means we don't need the return type to be `boolean`.